src/components/GameItem.js

```diff
@@ -2,15 +2,42 @@ import React, { Component } from 'react';
 import '../App.css';
 import ItemIcons from '../ItemIcons.js';
 import PropTypes from 'prop-types';
+import { spawn } from 'child_process';
+const randomObjProp = require('random-obj-prop');
```

Comment:
This line broke my build. When adding a new dependency, would should push the changes that are made to `package.json`. But in this case, it looks like you didn't actually end up using it so you could've just removed it.